lib/bufq.c

```diff
@@ -30,6 +30,7 @@
 #include "curl_memory.h"
 #include "memdebug.h"
 
+
```

Comment:
```suggestion
```

Reply:
I like double empty lines after the includes. Is this now forbidden?